README.md

```diff
@@ -1,6 +1,4 @@
-[![Python application](https://github.com/jernsting/GEM/actions/workflows/lint_and_test.yml/badge.svg)](https://github.com/jernsting/GEM/actions/workflows/lint_and_test.yml)
-[![Coverage Status](https://coveralls.io/repos/github/jernsting/GEM/badge.svg?branch=master)](https://coveralls.io/github/jernsting/GEM?branch=master)
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=jernsting_GEM&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=jernsting_GEM)
```

Comment:
I you'd like Coverage and Quality Gate Status badges, you need to add the project to https://coveralls.io and https://sonarcloud.io/ (free for open source projects).

For coveralls you also need to setup a repository secret.